RevenueCat/Scripts/Purchases.cs

```diff
@@ -872,6 +872,42 @@ public void SetAirshipChannelID(string airshipChannelID)
         _wrapper.SetAirshipChannelID(airshipChannelID);
     }
 
+    /**
+     * <summary>
+     * Subscriber attribute associated with the CleverTap ID for the user.
```

Comment:
NItpick. Since this is a setter, this should be `Sets subscriber attribute...` I think? Same for the others.

Reply:
good call, I updated it for all subs attributes